SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -1376,6 +1376,21 @@ \subsection{Const Correctness}
 \end{itemize}
 
 
+\subsection{Integer Type Specifiers}
+\label{subsec:IntegerTypeSpecifiers}
+
+Throughout the code base, the built-in type \code{unsigned int} is spelled exactly like that:
+``unsigned int'' (rather than just ``unsigned''). Other built-in integer types should in
```

Comment:
Thanks for the effort @N-Dekker. Why are some types left with quotes instead of consistently using the `\code{}` markup? To me, these are programming types/constructs not found in English language (as a whole) that deserve such highlighting.

Reply:
@jhlegarreta Thanks for asking. I did deliberately make a choice between `\code{...}` and double-quotes, each time. You see, this section specifically deals with how to _spell_ a certain type. 

 - `\code{unsigned int}` refers to the C++ type itself

 - "unsigned int" refers to the way we _spell_ it (according to our Style Guide).

Both spellings, "unsigned" and "unsigned int" denote the very same C++ type, `unsigned int`.

Hope that answers your question. 😃

Reply:
Still not convinced. If others think it is OK as it is, then it's maybe a very personal view of mine, and the conversation can be resolved.

Reply:
> If others think it is OK ...

Whichever.  I am willing to defer to the person doing the work.